src/Microsoft.Crank.RegressionBot/Program.cs

```diff
@@ -912,17 +912,23 @@ private static async IAsyncEnumerable<Regression> FindRegression(Source source)
                             Console.WriteLine($"Value: {currentValue}, benchmark runs on average every {(int)TimeSpan.FromSeconds(averageInSeconds).TotalMinutes} minutes with a stdev of {(int)TimeSpan.FromSeconds(standardDeviation).TotalMinutes} minutes. Intervals were: {String.Join(',', intervalsInSeconds)}");
                         }
 
+                        // We assume the benchmark is not running if it wasn't triggered for twice the expected delay.
+                        // The standard deviation could also be ignore here but since it's available let's take it into account.
```

Comment:
```suggestion
                        // The standard deviation could also be ignored here but since it's available let's take it into account.
```
NIT